docs/src/Guides/20 Scales.md

```diff
@@ -203,6 +203,36 @@ bot.grow_scale("Filename_here")
 bot.start("token")
 ```
 
-Finally, for the cool bit of Scales, reloading. Scales allow you to edit your code, and reload it, without restarting the bot.
+Additionally, for the cool bit of Scales, reloading. Scales allow you to edit your code, and reload it, without restarting the bot.
```

Comment:
"Additionally" here is... confusing. I think "Now" or something similar would be better.

Nitpick though.